endgame_postprocessing/model_wrappers/trachoma/README.md

```diff
@@ -0,0 +1,56 @@
+# Usage
+
+Having installed the endgame post processing pipeline, there exists functions for running
+the Trachoma pipeline.
+
+It can concatenate historic data and forward projections.
+
+```python
+from endgame_postprocessing.model_wrappers.trachoma import run_trach
+
+run_trach.run_postprocessing_pipeline(
+    input_dir=input_dir,
+    output_dir=output_path,
+    historic_dir=historic_dir,
+    historic_prefix="PrevDataset_Trachoma_",
+    start_year=2000,
+    stop_year=2041
+)
+```
+
+Here the `input_dir` is the directory of the forward projections *and* the
+Population file - see main [README](../../../README.md#iu-meta-data-file) for the format:
+This is expected to be in the following format:
```

Comment:
🚫 Because we changed the code to use `get_flat_regex` we don't require the hierachy of folders, see the [lf docs](https://github.com/NTD-Modelling-Consortium/endgame-postprocessing/tree/main/endgame_postprocessing/model_wrappers/lf) for the input file requirements